pipelines/broad/dna_seq/germline/joint_genotyping/UltimaGenomics/UltimaGenomicsJointGenotyping.changelog.md

```diff
@@ -1,3 +1,8 @@
+# 1.1.5
+2023-12-14 (Date of Last Commit)
+
+* Updated VETS filtering pipeline to GATK version 4.5.0.0.
```

Comment:
Does this affect outputs?

Reply:
It shouldn't. I updated the comment here.